### PR TITLE
feat: Add support for documentation generation in actions

### PR DESCRIPTION
**Pull request by Blond11516**
Adds support for generating function documentation in the action generator. Documentations include a (mandatory) link to the official discord doc as well as an optional custom section.

### CODE REVIEW

File: lib/mobius/actions.ex [hunk 1]
```diff
@@ -30,7 +30,11 @@ defmodule Mobius.Actions do
         name = endpoint.name
         param_names = Actions.get_arguments(endpoint)
         params_keyword = Actions.get_arguments_keyword(endpoint)
+        docs = Actions.get_docs(endpoint)
 
+        @doc """
+        #{docs}
+        """
```
Comment (ZeLarpMaster) [inline lib/mobius/actions.ex]: `@doc docs` doesn't do the same thing?

File: test/mobius/actions/message_test.exs [hunk 2]
```diff
@@ -28,7 +28,7 @@ defmodule Mobius.Actions.MessageTest do
       channel_id = random_snowflake()
       raw = message(channel_id: channel_id)
       url = Client.base_url() <> "/channels/#{channel_id}/messages"
-      mock(fn %{method: :post, url: ^url} -> json(raw) end)
+      mock(fn %{method: :post, url: ^url, body: _} -> json(raw) end)
```
Comment (ZeLarpMaster) [inline test/mobius/actions/message_test.exs]: why do we need `body: _`?

File: lib/mobius/actions.ex (hunk 1 above)
Comment (Blond11516) [reply lib/mobius/actions.ex]: Yup, it does :+1:

File: test/mobius/actions/message_test.exs (hunk 2 above)
Comment (Blond11516) [reply test/mobius/actions/message_test.exs]: I honestly don't remember :thinking: Feels like it had to with making sure we were passing the body to tesla, but tesla doesn't even provide a way to do a PATCH, POST or PUT without a body...

I've removed it.

**Review by ZeLarpMaster — APPROVED** [review]

**Pull request merged**